library/general/src/modules/Linuxrc.rb

```diff
@@ -211,6 +211,31 @@ def SaveInstallInf(root)
       true
     end
 
+    # Returns value of a given Linxurc key/feature defined on commandline
+    # and written into install.inf
+    #
+    # @param [String] key
+    # @return [String] value of a given key
+    def value_for(feature_key)
+      ReadInstallInf()
+
+      feature_key = polish(feature_key)
+      install_inf_features = deep_copy(@install_inf)
+
+      # Some values written on Linuxrc commandline are not known to Linuxrc
+      # Unless they are also mentioned as `PTOptions`, they will appear as "Cmdline" entry
+      install_inf_features.fetch("Cmdline", "").split.map do |cmdline_entry|
```

Comment:
If you do not need the result then replace `map` with `each`, this avoids creating an unused `Array`.

But the code still looks kind of ugly or weird for me, I'll try improving it a bit...